src/server/pfs/cmds/cmds.go

```diff
@@ -1371,6 +1378,8 @@ $ {{alias}} repo@branch -i http://host/path`,
 	}
 	copyFile.Flags().BoolVarP(&appendFile, "append", "a", false, "Append to the existing content of the file, either from previous commits or previous calls to 'put file' within this commit.")
 	copyFile.Flags().StringVar(&project, "project", project, "Project in which repo is located.")
```

Comment:
Can you add some more info in help text about how these 3 project flags interact. The behavior seems good for advanced usage, but I'm not sure I'd know how this fallback behavior works starting out. Maybe for each, add a line about what the default is like:
"Project in which the source repo is located. Uses the --project flag if not supplied"
and for --project "Project in which repo**s are** located. Uses the context default if not supplied."

I'm trying to keep it short, but also explicit, so feel free to change if you have a better idea for help text.

Reply:
I updated the command with examples as described above.

Reply:
Awesome!